simtools/corsika/corsika_config.py

```diff
@@ -498,7 +512,7 @@ def _write_seeds(self, file):
         for s in corsika_seeds:
             file.write(f"SEED {s} 0 0\n")
 
-    def get_input_file(self):
+    def get_input_file(self, use_multipipe=False):
         """
         Get the full path of the CORSIKA input file.
 
```

Comment:
Please add the Parameters section to the docstring

Reply:
Done